AK/IDAllocator.h

```diff
@@ -26,25 +26,24 @@
 
 #pragma once
 
-#include <stdlib.h>
 #include <AK/HashTable.h>
+#include <stdlib.h>
+#include <time.h>
 
 namespace AK {
 
 class IDAllocator {
 
 public:
-    IDAllocator() {}
-    ~IDAllocator() {}
+    IDAllocator() { srand(time(0)); }
```

Comment:
Calling `srand` means we change global state. What if a program wants to be reproducible and intentionally called `srand(MAGIC_BUT_CONSTANT_NUMBER)` initially? Do we care about such "weird" setups?